roles/ceph-common/defaults/main.yml

```diff
@@ -7,6 +7,20 @@
 
 fetch_directory: fetch/
 
+###############
+# PERMISSIONS #
+###############
+
+# Permissions for /etc/ceph configuration directory
+conf_directory_owner: root
+conf_directory_group: root
+conf_directory_mode: 644

```

Comment:
As long as you know the path to the file you are going to you are fine, if you need to run search on the directory then it won't work so well.